wgpu/backend/rs.py

```diff
@@ -863,18 +863,25 @@ def draw(self, vertex_count, instance_count, first_vertex, first_instance):
         )
 
     # wgpu.help('Buffer', 'Size64', 'renderencoderbasedrawindirect', dev=True)
-    def draw_indirect(self, indirect_buffer, indirect_offset):
-        raise NotImplementedError()
+    # def draw_indirect(self, indirect_buffer, indirect_offset):
+    #     raise NotImplementedError()
```

Comment:
what's that? draw indirect?

Reply:
Something like [this](https://www.khronos.org/opengl/wiki/GLAPI/glDrawArraysIndirect) I assume.